File: Week1_Homework/parser.rb
#Gil Dobrovinsky gilx0029 08/26/22

$break_line = "\n\n\n"


puts "#$break_line Parsing .txt File\n"
#Parse .txt
File.open("./SystemIntegration_individual/Week1_Homework/message.txt", "r") do |f|
    f.each_line do |line|
        puts line
    end
end

puts "#$break_line Parsing .csv File\n"



File.open("./SystemIntegration_individual/Week1_Homework/aboutme.csv", "r") do |f|
    f.each_line do |line|
        puts line
    end
end

puts "#$break_line Parsing .json File\n"


File.open("./SystemIntegration_individual/Week1_Homework/aboutme.json", "r") do |f|
    f.each_line do |line|
        puts line
    end
end

puts $break_line

File: Week1_Homework/parser.py
#Gil Dobrovinsky gilx0029 08/26/22
import os
import sys

breakline = "\n\n\n"

#Parse .txt
print(breakline, "Parsing .txt File\n")
with open(os.path.join(sys.path[0], "message.txt"), "r") as f:
    newline_break = ""
    for readline in f:
        line_strip = readline.strip()
        newline_break += line_strip
    print(newline_break)


print(breakline, "Parsing .csv File\n")

#Parse .csv
with open(os.path.join(sys.path[0], "aboutme.csv"), "r") as f:
    newline_break = ""
    for readline in f:
        line_strip = readline.strip()
        newline_break += line_strip
    print(newline_break)

print(breakline, "Parsing .json File\n")


#Parse .json
with open(os.path.join(sys.path[0], "aboutme.json"), "r") as f:
    newline_break = ""
    for readline in f:
        line_strip = readline.strip()
        newline_break += line_strip
    print(newline_break)

print(breakline)